cmd/experimental/kjobctl/pkg/builder/slurm_builder.go

```diff
@@ -193,6 +195,18 @@ func (b *slurmBuilder) build(ctx context.Context) (runtime.Object, []runtime.Obj
 	job.ObjectMeta.GenerateName = ""
 	job.ObjectMeta.Name = objectName
 
+	if len(b.priority) != 0 {
+		// check that priority exists
+		_, err = b.kueueClientset.KueueV1beta1().WorkloadPriorityClasses().Get(ctx, b.priority, metav1.GetOptions{})
+		if err != nil {
+			return nil, nil, err
+		}
```

Comment:
Can we move it to `validateGeneral()` function?

Reply:
Done.